tools/bazel_java_integration_test.bzl

```diff
@@ -22,6 +22,27 @@ load(
     "bazel_binaries",
 )
 
+def _bazel_java_integration_test_properties_impl(ctx):
+  properties = [
+      "bazel.version=" + ctx.attr.bazel_version,
+      "bazel.workspace=" + ctx.workspace_name,
+      "bazel.external.deps=" +
+      ",".join([d.short_path for d in ctx.files.external_deps]),
```

Comment:
@hchauvin this line collects all files of a filegroup from all filegroups? 
AFAICT each external dep is a bazel_external_dependency_archive which writes a filegroup with K entries.
Does the above collect all K entries from the V external dependencies?